pages/shields/oled.rst

```diff
@@ -0,0 +1,139 @@
+OLED Shield
+***********
+
+.. highlight:: python
+   :linenothreshold: 4
+
+.. figure:: /images/oled_shield_top.png
+   :width: 270
+
+   OLED Shield
+
+The `OLED Shield`_ is a Wemos D1 Mini shield that hosts a *tiny* 17mm 64x48
+monochrome OLED display.
+
+.. NOTE::
+
+    What is an *OLED*? It stand for Organic Light-Emitting Diode. It describes
+    the technology used to turn on or off pixels - dots - on the display. The
+    term is largely unimportant, just think of it like a tiny little TV where
+    you can control every pixel.
+
+Although small, this shield packs a punch! You can draw pixels, render lines
+and display text.
+
+The *integrated circuit* that makes the magic happen is an `SSD1306`_. It sits
+between the microcontroller and the raw display and allows us to send commands
+to make stuff appear. Because the SSD1306 is a common chip there is a driver
+for it built-in to MicroPython.
+
+Let's see how it works!
+
+.. _OLED Shield: https://wiki.wemos.cc/products:retired:oled_shield_v1.1.0
+.. _SSD1306: http://bleah.com
+
+Plug me in
+==========
+
+As with all the other shields, the first thing to do is plug the OLED shield
+into the microcontroller. Take care with orientation:
+
+.. figure:: /images/oled_shield_top.png
+
+   OLED Shield, plugged in correctly *[Update picture!]*
+
+Techy details, I squared C?
+===========================
+
+The SSD1306 is controlled by sending *I2C data* at it. What's I2C? It's not
+critical to know to use the OLED Shield but if you'd like to know more take
+a look at I2C_.
```

Comment:
I think it would be good to specify that it is a communications protocol to give a *slight* bit of context, but beyond that completely agree with just linking elsewhere

Reply:
Fixed in 9e2e99c